syntaxes/graphql.js.json

```diff
@@ -3,43 +3,40 @@
   "injectionSelector": "L:source -string -comment",
   "patterns": [
     {
-      "name": "taggedTemplates",
       "contentName": "meta.embedded.block.graphql",
-      "begin": "(Relay\\.QL|gql|graphql(\\.experimental)?)\\(?(`)",
+      "begin": "(?:Relay\\.QL|gql|graphql(\\.experimental)?)\\s*(?:<.*?>\\s*)?`",
```

Comment:
We now also allow for TypeScript generics here - `(?:<.*?>\\s*)?`.
That might come in handy if we add those in a future version of `graphql-tag`